### PR TITLE
compatible with thumbnail which url has no extension

### PR DESCRIPTION
**Pull request by elantion**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [x] Add unit tests for the feature.

### CODE REVIEW

**Comment by elantion** [comment]
ref: https://github.com/ant-design/ant-design/issues/9880

File: components/upload/__tests__/uploadlist.test.js [hunk 1]
```diff
@@ -264,7 +249,6 @@ describe('Upload List', () => {
     await delay(200);
     expect(wrapper.state().fileList[2].thumbUrl).not.toBeFalsy();
   });
-
```
Comment (afc163) [inline components/upload/__tests__/uploadlist.test.js]: Why delete those blanks?

Comment (elantion) [reply components/upload/__tests__/uploadlist.test.js]: After fighting 'LF' and 'CRLF', I forget reset those file. sorry. but it seem like pass lint after all.

Comment (afc163) [reply components/upload/__tests__/uploadlist.test.js]: You can reset these.

Comment (elantion) [reply components/upload/__tests__/uploadlist.test.js]: All right, mate. I add another commit to restore those blanks. Please review it. Thanks.

**Pull request merged**